common/alerts.js

```diff
@@ -48,6 +50,10 @@
                 scope.closeAlert = function(alert) {
                     AlertsService.deleteAlert(alert);
                 };
+
+                scope.login = function() {
```

Comment:
not sure if this should be defined here or if this should be a function that is passed along and attached to `scope`.

Reply:
Since we can now pass html with any callbacks in it to the alerts, maybe we should also add another parameter to `addAlert` which is an array of callbacks that we attach to scope and therefore can be called? I'm not sure what is the better solution here.